core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -362,6 +364,12 @@ private[serializer] object KryoSerializer {
     classOf[StorageLevel],
     classOf[CompressedMapStatus],
     classOf[HighlyCompressedMapStatus],
+    classOf[RoaringBitmap],
+    classOf[RoaringArray],
+    classOf[RoaringArray.Element],
```

Comment:
right, if we're bringing roaring bitmaps back, we need to address https://issues.apache.org/jira/browse/SPARK-11016.  There is also a comment about how to do this correctly here: https://issues.apache.org/jira/browse/SPARK-5949?focusedCommentId=14949819&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14949819.  we'll need to add test cases for this.